bedwars-plugin/src/main/java/com/andrei1058/bedwars/listeners/QuitAndTeleportListener.java

```diff
@@ -88,6 +88,10 @@ public void onLeave(PlayerQuitEvent e) {
             ss.cancel();
         }
 
+        if (SidebarService.getInstance() == null) {
+            return;
+        }
+
```

Comment:
I assume the NPE was because the sidebar service was  null, but then you should check if its not null and then remove it.  this way the other functions do get called aswell
`       
 if (SidebarService.getInstance() != null) {
            SidebarService.getInstance().remove(e.getPlayer());
 }
 `